### PR TITLE
Create multiple droplets

### PR DESCRIPTION
**Pull request by goodcode**
Initial support for creating multiple droplets with a single API call. The only thing that is missing is the ability to specify SSH keys.

### CODE REVIEW

**Comment by koalalorenzo** [comment]
Looks nice to me! Well done!
Can you write/add some the tests?

**Comment by goodcode** [comment]
I can try adding some tests. I would like to add the ssh-keys option as well because receiving automatic emails from Digital Ocean for each created droplet is kind of annoying.

File: digitalocean/Droplet.py [hunk 1]
```diff
@@ -460,14 +465,15 @@ def change_kernel(self, kernel, return_dict=True):
             return_dict
         )
 
-    def __get_ssh_keys_id_or_fingerprint(self):
+    @classmethod
+    def __get_ssh_keys_id_or_fingerprint(ssh_keys, token, name):
```
Comment (koalalorenzo) [inline digitalocean/Droplet.py]: self is missing

**Comment by koalalorenzo** [comment]
@goodcode yes, try to add some test exactly for this new method!

**Comment by goodcode** [comment]
I will try to add them next week. I'm sorry but I was busy the past few weeks.

**Comment by koalalorenzo** [comment]
Nice! Merging 👍 
I will test this during the weekend and release it ASAP :)

**Comment by goodcode** [comment]
Great, thanks!

**Pull request merged**